packages/client/components/TaskEditor/IntegrationBanner.tsx

```diff
@@ -0,0 +1,102 @@
+import styled from '@emotion/styled'
+import {Close} from '@mui/icons-material'
+import React from 'react'
+import useAtmosphere from '../../hooks/useAtmosphere'
+import {Integration} from '../../hooks/useIntegrationProviders'
+import useMutationProps, {MenuMutationProps} from '../../hooks/useMutationProps'
+import {PALETTE} from '../../styles/paletteV3'
+import {IntegrationProviders} from '../../types/constEnums'
+import AtlassianClientManager from '../../utils/AtlassianClientManager'
+import AzureDevOpsClientManager from '../../utils/AzureDevOpsClientManager'
+import GitHubClientManager from '../../utils/GitHubClientManager'
+import GitLabClientManager from '../../utils/GitLabClientManager'
+import JiraServerClientManager from '../../utils/JiraServerClientManager'
+import SlackClientManager from '../../utils/SlackClientManager'
+
+const SuggestIntegration = styled('div')({
+  display: 'flex',
+  flexDirection: 'row',
+  border: `1px solid ${PALETTE.SLATE_400}`,
+  borderRadius: '4px',
+  marginTop: '8px',
+  padding: '8px 6px'
+})
+
+const Link = styled('span')({
+  color: PALETTE.SLATE_800,
+  cursor: 'pointer',
+  textDecoration: 'underline',
+  fontWeight: 600,
+  ':hover, :focus, :active': {
+    color: PALETTE.SLATE_600
+  }
+})
+
+const SuggestionContent = styled('div')({
+  padding: 0
+})
+
+const CloseIcon = styled(Close)({
+  color: PALETTE.SLATE_600,
+  cursor: 'pointer'
+})
```

Comment:
+1 we're migrating to TailwindCSS, so if possible all new stuff should be done using Tailwind